app/controllers/pages_controller.rb

```diff
@@ -3,7 +3,7 @@ def home
   end
 
   def about
-    @members = Member.all
+    @members = Member.all.sort_by{|member| member.member_role}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:member_role` as an argument to `sort_by` instead of a block.